### PR TITLE
Version bump for amazon-chime-sdk-js@2.26.0

### PR DESCRIPTION
**Pull request by ltrung**
Version bump for amazon-chime-sdk-js@2.26.0

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
...
+    
+### Added
+    
+### Removed
+    
+### Changed
+    
+### Fixed
+    
 ## [Unreleased]
```
Comment (michhyun1) [inline CHANGELOG.md:18]: remove?

Comment (ltrung) [reply CHANGELOG.md:18]: Removed above right?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.26.0] - 2022-01-14
```
Comment (xuesichao) [inline CHANGELOG.md:8]: The order of `[Unreleased]` header and `[2.26.0] - 2022-01-14` header seems not right. Could you explain it a little bit?

Comment (ltrung) [reply CHANGELOG.md:8]: Oh good catch. Think Michael also pointed that out but I did not understand. Removed now.

**Review by michhyun1 — APPROVED** [review]

**Pull request merged**